utils/interface.py

```diff
@@ -0,0 +1,345 @@
+    )
+    benchmark.add_argument(
+        "-data_file", dest="data_file", help="A datafile that should be loaded."
+    )
+    benchmark.add_argument(
+        "-dfiles_num",
+        dest="dfiles_num",
+        default=1,
+        type=int,
+        help="Number of datafiles to input into database for processing.",
+    )
+    benchmark.add_argument(
+        "-iterations",
+        dest="iterations",
+        default=1,
+        type=int,
+        help="Number of iterations to run every query. Best result is selected.",
+    )
+    benchmark.add_argument(
+        "-dnd", default=False, type=str_arg_to_bool, help="Do not delete old table."
+    )
+    benchmark.add_argument(
+        "-dni",
+        default=False,
+        type=str_arg_to_bool,
+        help="Do not create new table and import any data from CSV files.",
+    )
+    benchmark.add_argument(
+        "-validation",
+        dest="validation",
+        default=False,
+        type=str_arg_to_bool,
+        help="validate queries results (by comparison with Pandas queries results).",
+    )
+    benchmark.add_argument(
+        "-import_mode",
+        dest="import_mode",
+        default="fsi",
+        help="you can choose: {copy-from, pandas, fsi}",
+    )
+    benchmark.add_argument(
+        "-optimizer",
+        choices=["intel", "stock"],
+        dest="optimizer",
+        default="intel",
+        help="Which optimizer is used",
+    )
+    benchmark.add_argument(
+        "-no_ibis",
+        default=False,
+        type=str_arg_to_bool,
+        help="Do not run Ibis benchmark, run only Pandas (or Modin) version",
+    )
+    benchmark.add_argument(
+        "-no_pandas",
+        default=False,
+        type=str_arg_to_bool,
+        help="Do not run Pandas version of benchmark",
+    )
+    benchmark.add_argument(
+        "-pandas_mode",
+        choices=["Pandas", "Modin_on_ray", "Modin_on_dask", "Modin_on_python"],
+        default="Pandas",
+        help="Specifies which version of Pandas to use: "
+        "plain Pandas, Modin runing on Ray or on Dask",
+    )
+    benchmark.add_argument(
+        "-ray_tmpdir",
+        default="/tmp",
+        help="Location where to keep Ray plasma store. "
+        "It should have enough space to keep -ray_memory",
+    )
+    benchmark.add_argument(
+        "-ray_memory",
+        default=200 * 1024 * 1024 * 1024,
+        help="Size of memory to allocate for Ray plasma store",
+    )
+    benchmark.add_argument(
+        "-no_ml",
+        default=False,
+        type=str_arg_to_bool,
+        help="Do not run machine learning benchmark, only ETL part",
+    )
+    benchmark.add_argument(
+        "-gpu_memory",
+        dest="gpu_memory",
+        type=int,
+        help="specify the memory of your gpu, default 16. "
+        "(This controls the lines to be used. Also work for CPU version. )",
+        default=16,
+    )
+
+
+def create_interface(omniscript_path, supported_tasks, supported_benchmarks):
```

Comment:
```suggestion
def create_cli(omniscript_path, supported_tasks, supported_benchmarks):
```
otherwise it's unclear which interface we're talking about

Reply:
done